apps/storybook-react/stories/Slider.stories.jsx

```diff
@@ -43,10 +46,32 @@ export const Examples = () => {
           onChange={onChange}
           ariaLabelledby="range-slider-label"
         />
-        <p>
+        <p style={{ marginTop: '1.5rem' }}>
```

Comment:
We should have a couple of utility classes available for these things 🤔  Or Styled System props maybe…

Reply:
Indeed... *kremt* tailwind